cmd/batsExecuteTests_generated.go

```diff
@@ -227,7 +227,7 @@ func batsExecuteTestsMetadata() config.StepData {
 						Name: "influx",
 						Type: "influx",
 						Parameters: []map[string]interface{}{
-							{"Name": "step_data"}, {"fields": []map[string]string{{"name": "bats"}}},
+							{"Name": "step_data", "fields": []map[string]string{{"name": "bats"}}},
```

Comment:
As this is unrelated, can you please extract the fix to a dedicated PR?

Reply:
here it is https://github.com/SAP/jenkins-library/pull/3362